aws/resource_aws_network_acl.go

```diff
@@ -118,6 +125,12 @@ func resourceAwsNetworkAcl() *schema.Resource {
 						"action": {
 							Type:     schema.TypeString,
 							Required: true,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
I agree it's common - one thing i saw in the committers channel the other day is that sometimes it's preferred to copy small pieces of code like this rather than need to vendor them - thoughts?